online.athn.browser.gnome.json

```
@@ -29,12 +29,11 @@
                 "cargo --offline fetch --manifest-path Cargo.toml --verbose",
                 "cargo --offline build --release --verbose",
                 "install -Dm755 target/release/reference-browser-gnome -t /app/bin/reference-browser-gnome",
```

this installs `/app/bin/reference-browser-gnome/reference-browser-gnome`

I doub't that's correct.

```suggestion
                "install -Dm755 target/release/reference-browser-gnome -t /app/bin",
```